jobs/build-arch.Jenkinsfile

```diff
@@ -1,14 +1,13 @@
 import org.yaml.snakeyaml.Yaml;
 
-def pipeutils, streams, official, repo, gp
+def pipeutils, streams, official, uploading
 def src_config_url, src_config_ref, s3_bucket
+def session
```

Comment:
In the commit message introducing this hunk: `s/no/now/`

Reply:
fixed